rasa/core/policies/embedding_policy.py

```diff
@@ -530,6 +531,7 @@ def _emebed_dialogue(self, dialogue_in: tf.Tensor) -> Tuple[tf.Tensor, tf.Tensor
         dialogue_transformed = self._tf_layers["transformer"](
             dialogue, 1 - tf.expand_dims(mask, axis=-1), self._training
         )
+        dialogue_transformed = tfa.activations.gelu(dialogue_transformed)
```

Comment:
You say `gelu` "doesn't make much of a difference". Why should we use it then?

Reply:
why not?